schema_automator/importers/tabular_import_engine.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+from dataclasses import dataclass
+from tempfile import NamedTemporaryFile
+from typing import List
+
+from linkml_runtime.linkml_model.meta import SchemaDefinition
+from schemasheets.schemamaker import SchemaMaker
+
+from schema_automator.importers.import_engine import ImportEngine
+import pandas as pd
+
+
+@dataclass
+class TableImportEngine(ImportEngine):
+    """
+    An ImportEngine that imports tabular data via schemasheets
+
+    """
+    element_type: str = None
+    parent: str = None
+    columns: List[str] = None
+
+    def convert(self, file: str) -> SchemaDefinition:
+        """
+        Converts one or more JSON files into a Schema
+
+        :param files:
+        :param kwargs:
+        :return:
+        """
+        df = pd.read_csv(file, sep='\t')
+        self.import_from_dataframe(df)
+
+    def import_from_dataframe(self, df: pd.DataFrame):
+        """
+        Imports a dataframe into a schema
+
+        :param df:
+        :return:
+        """
+        tf = NamedTemporaryFile()
```

Comment:
Windows tests fail because NamedTemporaryFile cannot be reopened on Windows. One possible solution is to open with delete=False:
`tf = NamedTemporaryFile(delete=False)  # with default (delete=True) tf cannot be reopened on Windows`

Reply:
and that worked too!